.releaserc

```diff
@@ -0,0 +1,146 @@
+{
+  "branches": [
+    "master",
+    {
+      "name": "next",
+      "prerelease": true
+    }
+  ],
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "angular",
+        "releaseRules": [
+          {
+            "type": "feat",
+            "release": "minor"
+          },
+          {
+            "type": "style",
+            "release": "patch"
+          },
+          {
+            "type": "refactor",
+            "release": "patch"
+          },
+          {
+            "scope": "skip",
+            "release": false
+          }
+        ],
+        "parserOpts": {
+          "noteKeywords": [
+            "BREAKING CHANGE",
+            "BREAKING CHANGES",
+            "BREAKING"
+          ]
+        }
+      }
+    ],
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        "preset": "conventionalcommits",
+        "presetConfig": {
+          "types": [
+            {
+              "type": "feat",
+              "section": "✨ New & Improved",
+              "hidden": false
+            },
+            {
+              "type": "refactor",
+              "section": "✨ New & Improved",
+              "hidden": false
+            },
+            {
+              "type": "style",
+              "section": "💄 Style Refinements",
+              "hidden": false
+            },
+            {
+              "type": "fix",
+              "section": "🛠 Fixes & Updates",
+              "hidden": false
+            },
+            {
+              "type": "perf",
+              "section": "🛠 Fixes & Updates",
+              "hidden": false
+            },
+            {
+              "type": "chore",
+              "section": "🛠 Fixes & Updates",
+              "hidden": false
+            },
+            {
+              "type": "docs",
+              "section": "📘 Tests & Docs",
+              "hidden": false
+            },
+            {
+              "type": "test",
+              "section": "📘 Tests & Docs",
+              "hidden": true
+            },
+            {
+              "type": "build",
+              "hidden": true
+            },
+            {
+              "type": "ci",
+              "hidden": true
+            }
+          ]
+        },
+        "parserOpts": {
+          "noteKeywords": [
+            "BREAKING CHANGE",
+            "BREAKING CHANGES",
+            "BREAKING"
+          ]
+        },
+        "writerOpts": {
+          "commitsSort": [
+            "subject",
+            "scope"
+          ],
+          "headerPartial": "## Version {{version}}"
+        }
+      }
+    ],
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogTitle": "Changelog\n==="
+      }
+    ],
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": true,
+        "tarballDir": "dist"
+      }
+    ],
```

Comment:
So wrt/[this comment](https://github.com/readmeio/api-explorer/pull/828/files#r453374712), I think you might be able to get away with just removing the options and letting it run through the default lerna publishing flow?

```suggestion
    "@semantic-release/npm",
```

Reply:
(I also spent a bunch of time reading through [this issue](https://github.com/semantic-release/semantic-release/issues/193) re: setting up semantic-release in a monorepo environment... I'm honestly still not quite clear on the problems this presents, having not spent much quality time with lerna. But it does seem like it'll may take a bit more thought to get this set up than I'd've expected. 😭)